### PR TITLE
Features

### PR DESCRIPTION
**Pull request by Tresor11**

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -0,0 +1,978 @@
...
+          <div class="col1  borde text-nowrap">World</div>
+          <div class="col1  borde text-nowrap">Business</div>
+          <div class="col1  borde text-nowrap">Tech & Science</div>
+          <div class="col1  borde text-nowrap">Culture</div>
+          <div class="col1  borde text-nowrap">NewsGeek</div>
+          <div class="col1  borde text-nowrap">Sports</div>
+          <div class="col1  borde text-nowrap">Health</div>
+          <div class="col1  borde text-nowrap">Opinion</div>
+          <div class="col2  text-nowrap px-2">Ventage <span class="px-2">search</span><i class="fa fa-search" aria-hidden="true"></i></div>
+        </div>
```
Comment (suretrust) [inline index.html]: These should be links, not just `div`.

File: index.html [hunk 2]
```diff
@@ -0,0 +1,978 @@
...
+  <link href="//netdna.bootstrapcdn.com/font-awesome/3.2.1/css/font-awesome.css" rel="stylesheet">
+  <link rel="stylesheet" href="style.css">
+  <script src="https://use.fontawesome.com/baa2c967f0.js"></script>
+  <title>Hello, world!</title>
+</head>
+
+<body>
+  <header>
+    <nav>
+      <div class="container-fluid ">
```
Comment (suretrust) [inline index.html:21]: This class should be `container`. Kindly take a look at the `newsweek.com` website, you'd notice some margin left and right.

**Review by suretrust — CHANGES_REQUESTED** [review]
Hi @Tresor11,

### Amazing job thus far. :raised_hands: Just a few suggestions to wrap this up:

- [ ] Please see my comments on your code.

- [x] Let's get rid of the horizontal scroll bar, this insinuates that there is some overflow in your code. Refer to the image below.

hor

- [x] Some `sections` or `section headers` are missing or named differently on your page, all sections must be present and named as is, on the `Newsweek's website`. This includes the featured stories section, top story section, opinion section, latest news section, more stories section, magazine section, editor's picks section, and featured slideshows section.

- [x] On medium screen, the text is supposed to wrap under the image. Refer to the images below:

**Original**
wrap

**Yours**
wrap2

- [x] On medium screen, under the latest news section, the image is more of a thumbnail than full image. Refer to the images below:

**Original**
img2

**Yours**
img

- [x] On medium screen, around the newsletter area, there are some margin to the left and right. Refer to the images below:

**Original**
pad

**Yours**
pad2

**Kindly submit for another code review once these issues have been fixed.** _Happy coding!_:tada:

**Comment by Anna-Myzukina** [comment]
### Pull Request under the review by Anna Muzykina (TSE)

**Comment by Anna-Myzukina** [comment]
### Hi Tresor!  
#### Wow, your project looks :+1: 
#### But need some Required Changes

Main content
* [x] It contains `Featured stories` section.
* [x] It contains `Top story` section.
* [x] It contains `Opinion` section.
* [x] It contains `Latest news` section.
* [x] It contains `More stories` section.
* [x] It contains `In the magazine` section.
* [x] It contains `Editor's picks` section.
* [x] It contains `Featured slideshows` section.
* [x] **It changes with RWD breakpoints.**

Please check responsiveness:

*** Also here button 'Sign in' ***
nw2

nw4

Please remember that your branch name should be descriptive (e.g., refactor-authentication, user-content-cache-key, make-retina-avatars), so that others can see what is being worked on. You can read more about it [here](https://microverse.pathwright.com/library/microverse-onboarding/86777/path/step/53987922/). I hope in the next project you can do this!

### Make changes and submit another code review request.

*If you have any questions, let me know on slack (@Anna-Muzykina) and I will be glad to clarify with you.*
*Happy coding 😄*

File: index.html [hunk 3]
```diff
@@ -0,0 +1,998 @@
...
+                the number of refugees admitted into the country to its lowest level in years.
+              </p>
+            </div>
+          </div>
+
+
+          <div class="col-12 col-md-6 col-lg-5 py-2 order-1 order-md-2 ">
+            <div class="col-12  mx-sm-1 two ">
+              <h6 class="font-weight-bolder text-muted">FEATURED STORIES</h6>
+              <img src="./images/bill-taylor-transcript-released.jpg" class="img-fluid w-100" alt="">
```
Comment (Anna-Myzukina) [inline index.html:126]: In line 125 should be "TOP STORY"
<h6 class="font-weight-bolder text-muted">TOP STORY</h6>

File: index.html [hunk 4]
```diff
@@ -0,0 +1,998 @@
...
+                          the number of refugees admitted into the country to its lowest level in years.
+                        </small>
+                      </p>
+                    </div>
+                  </div>
+                </div>
+
+                <div class="col-12 order-1 order-md-2">
+
+
```
Comment (Anna-Myzukina) [inline index.html:196]: Here should be "More stories"

File: index.html [hunk 5]
```diff
@@ -0,0 +1,998 @@
...
+                <div class="col-lg-3 col-2"><img src="./images/profil1.png" class="img-fluid w-100" alt=""></div>
+                <div class="col-lg-9 col-10">
+                  <p class="nowrap"> <span class="font-weight-bolder">Why Quid Pro Quo Works When Collusion
+                      Didn't</span><br>
+                    <span class="font-weight-bolder text-muted">BY NAVEED JAMALI</span></p>
+                </div>
+              </div>
+            </div>
+            <div class="row">
+              <h6 class="font-weight-bolder text-muted">LATEST NEWS</h6>
```
Comment (Anna-Myzukina) [inline index.html]: Line 378 Here should be "In the magazine"

File: index.html [hunk 6]
```diff
@@ -0,0 +1,998 @@
...
+        </div>
+      </div>
+    </section>
+    <section>
+      <div class="container-fluid p-0">
+        <div class="col-12">
+          <div class="row my-4 two">
+            <div class="col-6 m-auto">
+              <img src="./images/N.png" alt="">
+              <h6 class="font-weight-bolder d-inline">LATEST NEWS</h6>
```
Comment (Anna-Myzukina) [inline index.html]: Line 484 should be "Editor's picks"

**Comment by Anna-Myzukina** [comment]
## Project Approved image

### You can merge it and don't forget to fill the project completion form.

Please remember that your branch name should be descriptive (e.g., refactor-authentication, user-content-cache-key, make-retina-avatars), so that others can see what is being worked on. You can read more about it here. I hope in the next project you can do this!


*Happy coding 😄*

**Review by Anna-Myzukina — APPROVED** [review]

**Pull request merged**